pages/model/[uuid].tsx

```diff
@@ -78,6 +78,9 @@ function Model() {
   const open = Boolean(anchorEl)
   const [copyModelCardSnackbarOpen, setCopyModelCardSnackbarOpen] = useState(false)
   const [complianceFlow, setComplianceFlow] = useState<Elements>([])
+  const [showLastViewedWarning, setShowLastViewedWarning] = useState<boolean>(false)
```

Comment:
Super minor but the `<boolean>` type isn't needed as `useState` will infer the type from the initial value you've given it